src/resources/views/crud/filters/date_range.blade.php

```diff
@@ -1,4 +1,35 @@
 {{-- Date Range Backpack CRUD filter --}}
+
+@php
+    $filter->options['date_range_options'] = array_merge([
+		'timePicker' => false,
+    	'alwaysShowCalendars' => true,
+        'autoUpdateInput' => true,
+        'firsDay' => 0,
```

Comment:
```suggestion
        'firstDay' => 0,
```

Reply:
This is @pxpm 's to accept/reject, but I just wanted to ask @promatik - how did you make Github show the suggestion like this? So nice and clean!

![2020-10-30 07 44 34](https://user-images.githubusercontent.com/1032474/97701477-dfa36f00-1a83-11eb-901e-1559ddb2d4d2.gif)

Reply:
Yes, Github has that nice feature 😁
I usually go to the tab "Files Changed" to check everything changed on a PR, there you can do that;

![ezgif-4-c7e3f1f95478](https://user-images.githubusercontent.com/1838187/97702608-34a4ac80-1aa7-11eb-8528-83954119e7b8.gif)

You may do it for multi line too.

Reply:
TDIL 👍